src/components/lift-item/lifts-panel.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { LiftItem } from './lift-item';
 import { SpotlightValue } from '../spotlight-value';
 
-interface LiftsPanelProps {
+export interface LiftsPanelProps {
```

Comment:
Just a note to inform that I exported the `LiftPanelProps` since I need it and it was not exported before.